src/Normalizer/InstantArticleRssContentEntityNormalizer.php

```diff
@@ -26,7 +39,7 @@ public function normalize($data, $format = NULL, array $context = []) {
       'title' => $data->label(),
       'link' => $this->entityCanonicalUrl($data),
       'guid' => $data->uuid(),
-      'content:encoded' => parent::normalize($data, $format, $context),
+      'content:encoded' => $this->serializer->normalize($data, 'fbia', $context),
```

Comment:
Same format to RSS as well!

Reply:
This is a bit of a special case here. The `fbia` normalizer "normalizes" to a InstantArticle object, and the the `fbia_rss` encoder looks for that object and runs the objects `render` method to dump the FBIA markup. I see your point, this could probably be done better, and I'm breaking the interface, but I'd like to defer any further refactoring down this road to a later point.